rllib/core/rl_trainer/rl_trainer.py

```diff
@@ -353,3 +406,48 @@ def _make_distributed(self) -> MultiAgentRLModule:
             The distributed module.
         """
         raise NotImplementedError
+
+    def __get_param_ref(self, param: ParamType) -> Hashable:
```

Comment:
Can we move this function to the subclass? 

I get that they'll be seldom useful to users who are implementing, but I think that we can agree that logic belonging to the torch rl trainer should belong to the torch rl trainer, and the same for tensorflow. 

indexing into and adding to self._params and self._params_to_optim is really easy to do if you have this function.

Reply:
Oh ok, fair. 👍👍 so you want them to be just private. Not super private.

Reply:
yep yep

Reply:
ok this is also done.